charts/telegraf-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "telegraf-operator.fullname" . }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "telegraf-operator.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
Can the operator be horizontally scaled, or should we hard code this to 0?

Reply:
it already defaults to 3